packages/ai-constructs/API.md

```diff
@@ -49,6 +51,7 @@ type ConversationHandlerFunctionProps = {
         modelId: string;
         region?: string;
     }>;
+    outputStorageStrategy?: BackendOutputStorageStrategy<FunctionOutput>;
```

Comment:
This is similar to what data and auth construct do.

https://github.com/aws-amplify/amplify-backend/blob/27202b41269d029e7efc392d9f6340da36f12389/packages/auth-construct/src/types.ts#L472-L475